src/concrete/ml/quantization/quantized_module.py

```diff
@@ -598,9 +599,15 @@ def compile(
                 error to a default value.
             verbose (bool): Indicate if compilation information should be printed
                 during compilation. Default to False.
+            inputs_encryption_status (Optional[Sequence[str]]): encryption status ('clear',
```

Comment:
I would still mention that everything is encrypted if the parameter is set to None !

Reply:
It's mentionned in all the user facing APIs, I don't think many users will use this one tbh

Reply:
I mean it was mostly for making the docstring consistent with what this parameters does, as we now provide a way to chose what inputs to encrypt, to avoid confusion that we don't chose which ones are encrypted or not based on whatever other rules. But sure if no one else pushes for this then go for it